README.md

```diff
@@ -64,6 +64,10 @@ The `xcbeautify` binary will be installed at `Pods/xcbeautify/xcbeautify`
 
 ### Swift Package Manager
 
+Create a directory in the same location as the `xcodeproj` file, for example `Example`.  
```

Comment:
I was wrong.

`cd` is not necessary, because it is difficult to do it when combined with test execution.  
example: `xcodebuild [flags] | swift run -c release --package-path ./BuildTools xcbeautify`